sdks/js/packages/core/react/components/organization/plans/hooks/usePlans.tsx

```diff
@@ -59,7 +68,9 @@ export const usePlans = () => {
               cancel_url: cancel_url,
               success_url: success_url,
               subscription_body: {
-                plan: planId
+                plan: planId,
+                skip_trial: !isTrial,
+                cancel_after_trial: config?.billing?.cancelAfterTrial || false
```

Comment:
Should we set the default behaviour to cancel by default instead of setting `false`?
We've mainly built and tested frontier to work with a single paid plan. The default behaviour here might introduce untested scenarios.

Reply:
Bump